hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java

```diff
@@ -869,7 +871,9 @@ public void testNearCacheIdleRecordsEvicted() {
         stats = map.getLocalMapStats().getNearCacheStats();
 
         assertEquals("as the hits are not equal, the entries were not cleared from near cash after MAX_IDLE_SECONDS",
-                hitsBeforeIdleExpire, stats.getHits(), size);
```

Comment:
The `size` was acting as `delta` parameter for the assert, so we basically disabled the assert, although the hit counters were different.